helm_deploy/apply-for-legal-aid/templates/_envs.tpl

```diff
@@ -304,12 +304,12 @@ env:
   - name: LEGAL_FRAMEWORK_API_HOST
```

Comment:
@RoseSAK can you also remove these values from secrets.yaml now?

Reply:
I was planning on doing that separately tomorrow - I am following this guidance on Confluence: https://dsdmoj.atlassian.net/wiki/spaces/ATPPB/pages/5154930715/Migrate+from+git-crypt+to+AWS+secret+manager which says to wait a couple of days, but Colin said it was probably okay to just wait a few hours/overnight